.gitignore

```diff
@@ -45,6 +45,8 @@ jspm_packages
 # The generated index.js
 quickstart/public/index.js
 examples/mediadevices/public/index.js
+examples/localvideosnapshot/public/index.js
```

Comment:
I guess we didn't implement this for `mediadevices`, but i was thinking `local-video-snapshot` might read better in the URL and project structure. Also, not sure that we need to call out "local"—can we remove this prefix?